app/src/main/java/org/oppia/android/app/home/recentlyplayed/RecentlyPlayedTitleEnum.kt

```diff
@@ -0,0 +1,16 @@
+package org.oppia.android.app.home.recentlyplayed
+
+import android.content.Context
+import androidx.annotation.StringRes
+import org.oppia.android.R
+
+/** Represents different titles for RecentlyPlayedActivity used by screen readers. */
+enum class RecentlyPlayedTitleEnum(@StringRes private var title: Int) {
+  RECENTLY_PLAYED_STORIES(R.string.recently_played_stories),
```

Comment:
Why isn't this using recently_played_activity_title?

Reply:
Correct. Used `recently_played_activity_title `